internal/services/datafactory/data_factory_linked_service_sftp_resource.go

```diff
@@ -103,31 +121,43 @@ func resourceDataFactoryLinkedServiceSFTP() *pluginsdk.Resource {
 				},
 			},
 
-			"skip_host_key_validation": {
-				Type:     pluginsdk.TypeBool,
-				Optional: true,
+			"private_key_passphrase": {
+				Type:          pluginsdk.TypeString,
+				Optional:      true,
+				Description:   "Specify the password to decrypt the private key if the key file or the key content is password protected",
```

Comment:
nit - we don't do anything with these (currently), and you've only added them for some items, so might be best to just leave them out in future and just put them in the docs.